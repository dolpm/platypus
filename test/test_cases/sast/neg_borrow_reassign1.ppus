pipe main [] |> unit {
    int w = 5;
    mut &int a = &w;
    {
        int x = 4;
        {
            a = &x;
        }
    }
    

    |> ();
}