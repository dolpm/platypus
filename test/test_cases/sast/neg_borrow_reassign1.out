Fatal error: exception Failure("Immutable borrow (&'_ int : &(int : x)) has larger lifetime than a, so a cannot be rebound.")
